include/hse/hse.h

```diff
@@ -670,12 +653,11 @@ hse_kvs_param_get(
  */
 /* MTF_MOCK */
 hse_err_t
-hse_kvs_prefix_delete(
-    struct hse_kvs *     kvs,
-    unsigned int         flags,
-    struct hse_kvdb_txn *txn,
-    const void *         pfx,
-    size_t               pfx_len);
+hse_kvs_prefix_delete(struct hse_kvs *kvs,
+        unsigned int flags,
```

Comment:
Ideally this would not be continued with 8 spaces.

Reply:
Cleaner if all the args line up, regardless of wherever the first arg starts...

Reply:
fixed